src/app/app.component.ts

```diff
@@ -73,25 +107,6 @@ export class AppComponent implements AfterViewInit, OnDestroy {
     ).subscribe();
   }
 
-  ngAfterViewInit(): void {
-    this._ipcBus.on(Message.ChangeTheme, (_: any, theme: string) => {
```

Comment:
Is there any particular reason you decide to move it to the constructor?

Reply:
There is no particular reason I would say. Just I prefer to keep all subscriptions which should live until component is not destroyed, in the constructor. I'm trying to use lifecycle hooks only when it's really required.

Reply:
Since this is `ngAfterViewInit` I'd be worried to move the logic to the constructor. Maybe there's something dependent on the view initialization? Could we revert this back?

Reply:
Okay sure, I'm gonna check and revert it.